massa-execution-worker/src/interface_impl.rs

```diff
@@ -512,6 +691,48 @@ impl Interface for InterfaceImpl {
         Ok(keys)
     }
 
+    /// Get the operation datastore keys (aka entries).
+    /// Note that the datastore is only accessible to the initial caller level.
+    ///
+    /// # Returns
+    /// A list of keys (keys are byte arrays) that match the given prefix
+    fn get_op_keys_wasmv1(&self, prefix: &[u8]) -> Result<Vec<Vec<u8>>> {
+        // compute prefix range
+        let prefix_range = if !prefix.is_empty() {
+            // compute end of prefix range
+            let mut prefix_end = prefix.to_vec();
+            while let Some(255) = prefix_end.last() {
+                prefix_end.pop();
+            }
```

Comment:
Note: to update once this is merged: https://github.com/massalabs/massa/pull/4152

Reply:
Done